packages/framework/src/Migrations/Version20191121145700.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Shopsys\FrameworkBundle\Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Shopsys\MigrationBundle\Component\Doctrine\Migrations\AbstractMigration;
+
+class Version20191121145700 extends AbstractMigration
+{
+    /**
+     * @param \Doctrine\DBAL\Schema\Schema $schema
+     */
+    public function up(Schema $schema): void
+    {
+        $contactFormMainTextCount = $this->sql('SELECT COUNT(*) FROM setting_values WHERE name = \'contactFormMainText\' AND domain_id = 1;')->fetchColumn(0);
+        if ($contactFormMainTextCount <= 0) {
+            $this->sql('INSERT INTO setting_values (name, domain_id, value, type) VALUES (\'contactFormMainText\', 1, :text, \'string\')', [
+                'text' => 'Shopsys Framework - Contact form',
+            ]);
```

Comment:
I haven't do anything with it because I am not sure, how to solve it. From my point of view, it should be in a migration, because after upgrade, we want to be sure that new setting will be added to the application. Even for application which are in production.
And I guess, It could cause application failing, when you try to enter the new section `Contact page` in administration because of the missing setting
In upgrade notes I've added step, to move all content from previous page to the new contact page.

Reply:
Okay, I agree with migration and adding the upgrade note to move content seems good to me.

But when I load database (`./phing db-demo`) in 9.0 branch, check out this PR and run migrations (`./phing db-migrations`), `contactFormMainText` is added into settings only for the first domain. Resulting in 500 error on /admin/contact-form/ page when I select second domain (`Setting value with name "contactFormMainText" for domain with ID "2" not found.`)

Reply:
Hmmm, I know that fact. I was thinking about it for a while. I can't find the best solution

when you run just `./phing db-migrations` it create setting only for the first domain. There exists target `./phing domains-data-create` which fills missing settings for other domains. In production this command run after `db-migration`. 

To solve that problem I added to that migration `MultidomainMigrationTrait` and loop through all domains. It solves problem with running only `./phing db-migrations`
But on the other hand when you run `db-demo` in the time of migration running there exists only one domain. So looping through domains will have no effect. It counts on `domains-data-create` to do its work  

So fixed

Reply:
👍 nice!